test/Interop/Cxx/class/conforms-to-errors.swift

```diff
@@ -15,6 +15,10 @@ struct  __attribute__((swift_attr("conforms_to:Mod.X"))) CModInv {};
 struct  __attribute__((swift_attr("conforms_to:SwiftTest.X"))) CX {};
 struct  __attribute__((swift_attr("conforms_to:SwiftTest.A"))) CA {};
 struct  __attribute__((swift_attr("conforms_to:SwiftTest.B"))) CB {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.A"))) \
+    __attribute__((swift_attr("conforms_to:SwiftTest.B"))) ConformsToValidAProtocolButAlsoStructB {};
+struct  __attribute__((swift_attr("conforms_to:SwiftTest.A"))) \
```

Comment:
Sorry for the late comment, I wonder if we want to have a test where the user conforms a struct to the same protocol twice just to make sure there are no crashes or unintended behavior. After that, this patch looks good to me.